region/maridia/outer.json

```diff
@@ -120,6 +120,117 @@
               "openEnd": 1,
               "devNote": "The runway is longer with the tube intact, but we'll leave this out because it's gone forever once the tube's broken."
             }
+          ],
+          "leaveWithGMode": [
+            {
+              "leavesWithArtificialMorph": false,
+              "strats": [
+                {
+                  "name": "Carry G-Mode Through Tube (Right to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }}
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Through Tube (Bottom to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken"
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Through Tube (Top to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken",
+                    {"or":[
```

Comment:
These items should be in the bottom to left strat?

Reply:
Because of the way the logic works, this strat technically starts at the left door. So in order to have the gmode setup work, you also need to ensure you can get to the door that has the setup. So these are here to describe how to get to the top door (getting to the bottom and back is free)